addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -1694,6 +1694,21 @@ body.editor_enable.editor_has_snippets {
                 background-color: $o-we-sidebar-bg;
             }
         }
+
+        div.o_we_invisible_element_is_root {
+            padding-bottom: 3px;
+        }
+
+        ul.o_we_list_invisible_element {
```

Comment:
2 things:
- I think that naming this new class `o_we_list_XXX` is bad because we_list is an existing widget that has nothing to do with the invisible elements
- Do you really need this class ? Why ? :thinking: